### PR TITLE
docs(misconfiguration): added instruction for misconfiguration detection

### LINKED ISSUE #1301: --download-db-only for trivy config
Could we replicate this feature/behaviour from "trivy image" where you can use --download-db-only to install the db/files for vuln scanning up front, but for "trivy config" when wanting to check for misconfigurations.

If gchr is down/unstable, or for offline usage, this would be useful.

Thanks!

### PR DESCRIPTION
**Pull request by DmitriyLewen**
added instruction for misconfiguration detection
Closes #1301

### CODE REVIEW

File: docs/advanced/air-gap.md [hunk 1]
```diff
@@ -47,11 +50,69 @@ $ rm trivy-offline.db.tgz
...
+$ rm bundle.tar.gz
+```
+
+In an air-gapped environment it is your responsibility to update policies on a regular basis, so that the scanner can detect recently-identified misconfigurations. 
+
+### Run Trivy with --skip-policy-update option
+In an air-gapped environment, specify `--skip-policy-update` so that Trivy doesn't attempt to download the latest misconfiguration policies.
+
+```
+$ trivy --skip-policy-update
```
Comment (knqyf263) [inline docs/advanced/air-gap.md:114]:
```suggestion
$ trivy conf --skip-policy-update /path/to/conf
```

File: docs/advanced/air-gap.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
 # Air-Gapped Environment
 
-Trivy can be used in air-gapped environments.
+Trivy can be used in air-gapped environments.
```
Comment (knqyf263) [inline docs/advanced/air-gap.md:3]: Needed?

Comment (DmitriyLewen) [reply docs/advanced/air-gap.md]: it's removed

**Review by knqyf263 — APPROVED** [review]
:shipit:

**Pull request merged**